### DISCUSSION TITLE
WebUI4CSharp available

### DISCUSSION THREAD

**Discussion by salvadordf** [webui-dev/webui#312] [General]
Hi,

Thank you very much for adding WebUI4Delphi to the list of WebUI wrappers in this repository.

I just wanted to let you know that I released a new WebUI wrapper for C# called [WebUI4CSharp](https://github.com/salvadordf/WebUI4CSharp). 

It includes console, WinForms and WPF demos.

**Comment by AlbertShown** [comment]
Wonderful! Thank you for this second wrapper 👍

**Comment by AlbertShown** [comment]
I guess `CSharp-WebUI` is the correct name to be consistent with the other wrappers' names. Also, the Delphi one should be `Delphi-WebUI`. And the importing library name should be `webui` as soon as possible. However, it's not mandatory, but it will be really good to keep the names as similar as possible.

You can update the name if you have free time. Otherwise, you can keep it as it is.
